config/cdConfig.js

```diff
@@ -20,6 +20,11 @@ const crawlerStoreProvider = config.get('CRAWLER_STORE_PROVIDER') || 'cd(file)'
 const maxRequeueAttemptCount = config.get('CRAWLER_MAX_REQUEUE_ATTEMPTS') || 5
 const fetchedCacheTtlSeconds = config.get('CRAWLER_FETCHED_CACHE_TTL_SECONDS') || 60 * 60 * 8 //8 hours
 
+function getLicenseeParallelism() {
+  const num = Number(config.get('CRAWLER_LICENSEE_PARALLELISM') || process.env.CRAWLER_LICENSEE_PARALLELISM)
+  return num > 0 ? num : 10;
+}
+
```

Comment:
I like that you added a validation that the config is a positive number.  It would be better to generalize this function so other configs that require a positive num can be refactored to use it when someone has time.  

```suggestion
function getPositiveNum(configName, defaultValue) {
  const num = Number(config.get(configName))
  return num > 0 ? num : defaultValue;
}

```

NOTE: I removed `process.env. CRAWLER_LICENSEE_PARALLELISM`.  None of the other configs come from env vars.  To introduce this approach would require the other configs to support env vars as well.  Can you say why you decided to go this route?

Reply:
I added `process.env`, since it's also used for `CRAWLER_SCANCODE_PARALLELISM`.
That seems to be more of an outlier, so I think reading from `config` makes more sense.